src/runtime/storage/crdt-collection-model.ts

```diff
@@ -122,16 +123,16 @@ export class CrdtCollectionModel {
     return effective;
   }
 
-  // [{id, value, keys: []}]
+  // Serialized ModelEntry is [{id, value, keys: []}]
```

Comment:
This seems redundant given we've the type definition now?

Reply:
agreed. removed comment.